jdxsmith/rancid-tomatillos
# Testing/add asynchronous tests

#### What’s this PR do?
- Refactored our movie data API call to happen in `Poster.js` rather than `App.js`
- Fixed infinite rendering issue
- Updated testing suite to account for the new API call structure
- Added asynchronous testing
- Used a React hook to update a state in our `Poster.js`

#### Where should the reviewer start?
- At the top of each changed file

#### How should this be manually tested?
- By checking the console in the app and ensuring that no errors are occurring
- By rinning `npm start` in the terminal to check that all tests pass

#### Any background context you want to provide?
- Experimented with React hooks rather than creating a class component

#### What are the relevant tickets?
- Closes #21 
- Closes #22 
- Closes #23
